src/eth/rpc.ts

```diff
@@ -51,7 +51,7 @@ export async function getSnapshotRound ():Promise<BigNumber> {
         if (process.env.SNAPSHOT_ROUND != "") {
             return BigNumber.from(process.env.SNAPSHOT_ROUND)
         } else {
-            return await roundsManager.lipUpgradeRound(BigNumber.from(36))
+            return (await roundsManager.lipUpgradeRound(BigNumber.from(36))).sub(BigNumber.from(1))
```

Comment:
We could use `lipUpgradRound(BigNumber.from(52))` here if it is correctly set , this is more inline with [the contracts ](https://github.com/livepeer/protocol/blob/e7aec27acb0eb9542f64123d2ad89b398cc2643c/contracts/bonding/BondingManager.sol#L463).

I can make the change if you'd like

Reply:
SGTM feel free to push the change & merge.